README.md

```diff
@@ -261,7 +261,9 @@ Brainstem parses the request params and supports the following:
   `per_page` is 20 and the default `:max_per_page` is 200.
 * Brainstem supports a concept called "only queries" which allow you to request a specific set of records by ID, kind of like
   a batch show request. These queries are triggered by the presence of the URL param `"only"` with a comma-seperated set
-  of one or more IDs, for example `"only=1,5,7"`.
+  of one or more IDs, for example `"only=1,5,7"`. Please note that default filters are still applied to only queries, so you will receive
```

Comment:
Might be good to put the only in "applies to only queries" in backticks.

> applies to `only` queries

Just to make it a bit more clear.